cmake/external_onnx.cmake

```diff
@@ -32,6 +32,7 @@ set(ONNX_GIT_BRANCH rel-${ONNX_VERSION})
 
 add_definitions(-DONNX_BUILD_SHARED_LIBS=ON)
 add_definitions(-DONNX_ML=ON)
+add_definitions(-DONNX_NAMESPACE=onnx)
```

Comment:
We should customize this namespace, not use the default `onnx`. This should be done in a separate PR.